code/code/disc/disc_psionics.cc

```diff
@@ -366,19 +365,17 @@ void TBeing::doPShout(const char *msg){
   } else {
     learnFromDoing(SKILL_PSITELEPATHY, SILENT_NO, 0);
 
-    mud_str_copy(garbed, garble(NULL, msg, Garble::SPEECH_SHOUT, Garble::SCOPE_EVERYONE), 256);
+    mud_str_copy(message, msg, 256);
```

Comment:
What's the point of this?

Reply:
I was trying to stick with the existing code as much as possible.  If we move from character arrays as suggested above, I suspect this will also change/go away.